Articles on API security, server security and cyber security (#6001)
kamranahmedse/developer-roadmap: src/data/roadmaps/backend/content/server-security@TZ0BWOENPv6pQm8qYB8Ow.md

- Enable two-factor authentication: Two-factor authentication adds an extra layer of security to your accounts by requiring a second form of authentication, such as a code sent to your phone, in addition to your password.

Also learn about OpenSSL and creating your own PKI as well as managing certs, renewals, and mutual client auth with x509 certs

> 
> Few Resources : <br>
> - [@article@How to Defend Against Server-Side Request Forgery](https://www.freecodecamp.org/news/defending-against-ssrf-attacks/)

Reviewer: ```suggestion

Learn more from the following resources:

- [@article@How to Defend Against Server-Side Request Forgery](https://www.freecodecamp.org/news/defending-against-ssrf-attacks/)
```
